custom/icds_reports/models/aggregate.py

```diff
@@ -563,6 +565,22 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateTHRForm(models.Model, AggregateMixin):
+    state_id = models.TextField()
+    supervisor_id = models.TextField()
+    awc_id = models.TextField()
+    month = models.DateField()
+    image_count = models.IntegerField(help_text='Count of Images clicked per awc')
+
+    objects = CitusComparisonManager()
+
+    class Meta(object):
+        db_table = AGG_THR_V2_TABLE
```

Comment:
I think that I started this convention, but I think that was a mistake. We should be able to use django's generated name and use something like `AggregateTHRForm._meta.db_table` to reference it elsewhere.

Though there is some value in having all helpers use the same interface

Reply:
yeah, I think that would be better If we make all the indicator same as that.
Do you think its needs to be changed here?

Reply:
No I think tihs is fine to leave it as is